### ISSUE TITLE
Preventing 301 redirects on URLs 

### ISSUE THREAD

**Issue by larsroettig** [narative/gatsby-theme-novela#382]
# Prerequisites
* [ ] Have you searched the issue tracker for similar issues (either past or present)?
* [ ] Can you reproduce the issue consistently?
* [ ] Are you reporting to the correct repository?
* [ ] Is this something you can **debug and fix**? Send a pull request! Bug fixes and documentation fixes are welcome.

# Description
1. Open Browser Console
2. Enter https://novela.narative.co/Why-we-built-a-company-before-building-a-product

URLs with no trailing slash lead to an immediate 301 redirect to the page with a trailing slash.

This has a performance cost and implications for SEO.

### Version of Novela you are using
www@1.6.1

### Number of posts you have and which data source
* [ x ] Local
* [ ] Contentful

### Describe the bug
_A clear and concise description of what the bug is._

### Expected behavior
_A clear and concise description of what you expected to happen._

### Screenshots
_If applicable, add screenshots to help explain your problem._

### Browser
_Browser [e.g. Chrome, Safari, iOS Safari]_

### Additional context
_Add any other context about the problem here._

**Comment by slorber** [comment]
Yes, I have that on my blog too and definitively would like it to be fixed.

Any idea what's the cause?

**Comment by larsroettig** [comment]
@slorber I don't know I added additional slash in my own theme. The base issue could be  https://github.com/gatsbyjs/gatsby/issues/9207

My fix for own theme:
https://github.com/larsroettig/larsroettig.dev/blob/c08d5c5c16de4428af62a20c1ac708a6650795f1/apps/larsroettig.dev/gatsby/node/createPages.js#L99
